pong-game/game-service/app/game_service/game_room.py

```diff
@@ -0,0 +1,237 @@
+import math
+import json
+import asyncio
+import httpx
+import logging
+from asgiref.sync import async_to_sync
+
+CANVAS_WIDTH = 800 #original value is 800, change it in case I fucked it up for tests
+CANVAS_HEIGHT = 600
+PADDLE_HEIGHT = 100
+PADDLE_WIDTH = 15
+BALL_RADIUS = 10
+
+logger = logging.getLogger(__name__)
+class Player():
+    def __init__(self, player_id, side, canvas_width, canvas_height):
+        self.player_id = player_id
+        if side == 'left':
+            self.x = canvas_width * 0.10
+        elif side == 'right':
+            self.x = canvas_width * 0.90
+        self.y = canvas_height * 0.30
+        self.speed = 0
+        self.score = 0
+
+    def get_speed(self):
+        return self.speed
+
+    def set_speed(self, value):
+        self.speed = value
+
+class Ball():
+    def __init__(self, canvas_width, canvas_height):
+        self.canvas_width = canvas_width
+        self.canvas_height = canvas_height
+        self.x = canvas_width * 0.5
+        self.y = canvas_height * 0.5
+        self.speedX = 5
+        self.speedY = 5
+        self.radius = BALL_RADIUS
+
+class GameRoom():
+    def __init__(self, room_id, player_channel, players):
+        self.room_id = "lobby_" + room_id
+        self.player_channel = player_channel
+        self.players = {
+                0: Player(players[0], 'left', CANVAS_WIDTH, CANVAS_HEIGHT),
+                1: Player(players[1], 'right', CANVAS_WIDTH, CANVAS_HEIGHT)
+        }
+        self.canvas_width = CANVAS_WIDTH
+        self.canvas_height = CANVAS_HEIGHT
+        self.ball = Ball(self.canvas_width, self.canvas_height)
+        self.running = True
```

Comment:
Unused variable, remove it or find a use for it for the consumer cleanup for instance